lib/commands/file-movement.js

```diff
@@ -321,7 +321,74 @@ async function pullFromRealDevice (device, remotePath, isFile) {
     }
   } catch (e) {
     if (e.message.includes(OBJECT_NOT_FOUND_ERROR_MESSAGE)) {
-      throw new Error(`Path '${remotePath}' doesn't exists on the device`);
+      throw new Error(`Path '${remotePath}' does not exist on the device`);
+    }
+    throw e;
+  } finally {
+    service.close();
+  }
+}
+
+/**
+ * Remove the file or folder from the device
+ *
+ * @param {Object} device - The device object, which represents the device under test.
+ *                          This object is expected to have the `udid` property containing the
+ *                          valid device ID.
+ * @param {string} remotePath - The path to a file or a folder, which exists in the corresponding application
+ *                              container on Simulator. Use
+ *                              @<app_bundle_id>:<optional_container_type>/<path_to_the_file_or_folder_inside_container>
+ *                              format to pull a file or a folder from an application container of the given type.
+ *                              Possible container types are 'app', 'data', 'groups', '<A specific App Group container>'.
+ *                              The default type is 'app'.
+ */
+async function deleteFromSimulator (device, remotePath) {
+  let pathOnServer;
+  if (CONTAINER_PATH_PATTERN.test(remotePath)) {
+    const { bundleId, pathInContainer: dstPath } = await parseContainerPath(remotePath,
+      async (appBundle, containerType) => await getAppContainer(device.udid, appBundle, null, containerType));
+    log.info(`Parsed bundle identifier '${bundleId}' from '${remotePath}'. ` +
+      `Will get the data from '${dstPath}'`);
+    pathOnServer = dstPath;
+  } else {
+    const simRoot = device.getDir();
```

Comment:
this may potentially damage Simulator FS. Should we rather limit absolute paths, to only known subfolders?

Reply:
I'm not sure that it's much of a problem. If people mess things up, oh well?

Reply:
ok, only full reset will restore the Simulator in such case